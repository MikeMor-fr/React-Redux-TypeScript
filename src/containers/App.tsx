import React, { Component } from 'react';
import HomePage from '../components/HomePage/HomePage';

class App extends Component {
  render(): JSX.Element {
    return <HomePage />;
  }
}

export default App;
