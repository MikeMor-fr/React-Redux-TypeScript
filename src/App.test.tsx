import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('Basic test', () => {
  it('Renders the App component', () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
  });
});
