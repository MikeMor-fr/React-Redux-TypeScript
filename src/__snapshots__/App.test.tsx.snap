// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic test Renders the App component 1`] = `ShallowWrapper {}`;
